automation/check-patch.e2e-multus-functests.sh

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env bash
+
+set -xeuE
+
+# This script should be able to execute multus
+# functional tests against Kubernetes cluster with
+# CNAO built with latest changes, on any
+# environment with basic dependencies listed in
+# check-patch.packages installed and docker running.
+#
+# yum -y install automation/check-patch.packages
+# automation/check-patch.e2e-multus-functests.sh
+
+teardown() {
+    rm -rf "${TMP_COMPONENT_PATH}"
+    make cluster-down
+}
+
+function __get-tools() {
+    mkdir -p bin
+
+    echo "install jq"
+    curl -Lo ./bin/jq https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64
```

Comment:
maybe check if no jq already ? 
else it would just curl it each run

Reply:
Done